src/lib.rs

```diff
@@ -308,66 +308,62 @@ If you want the server to replicate an entity into a client entity that was alre
 This can be useful for certain types of game. For example, spawning bullets on the client immediately without
 waiting on replication.
 
-### "Blueprints" pattern
+### Required components
 
-The idea was borrowed from [iyes_scene_tools](https://github.com/IyesGames/iyes_scene_tools#blueprints-pattern).
 You don't want to replicate all components because not all of them are
 necessary to send over the network. For example, components that are computed based on other
-components (like [`GlobalTransform`]) can be inserted after replication.
-This can be easily done using a system with query filter.
-This way, you detect when such entities are spawned into the world, and you can
-do any additional setup on them using code. For example, if you have a
-character with mesh, you can replicate only your `Player` and [`Transform`] components and insert
-necessary components after replication. To avoid one frame delay, put
-your initialization systems in [`ClientSet::Receive`]:
+can be automatically inserted after replication thanks to the Bevy's required components.
+For components that require world access you can create a special system that inserts such
+components after entity spawn. To avoid one frame delay, put your initialization systems
+in [`ClientSet::Receive`]:
```

Comment:
```suggestion
can be automatically inserted after replication thanks to Bevy's required components.
For components that require world access you can create a special system that inserts such
components after entity spawn. To avoid one frame delay, put your initialization systems
in [`ClientSet::Receive`]:
```